frontend/src/components/App/Toast.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component, PropTypes } from 'react'
+
+class Toast extends Component {
+  static propTypes = {
+    message: PropTypes.string
+  }
+
+  render () {
+    const html = {__html: this.props.html}
+    return <p id="toast" className="toast" dangerouslySetInnerHTML={html} />
+  }
+}
+
+export default Toast
```

Comment:
FYI currently i'm using 8000ms as the default timeout, and i really want to change it. not sure to what. Maybe 5000ms